aggregator/rpc_server.go

```diff
@@ -41,6 +41,8 @@ type RpcServer struct {
 	ethrpc *ethclient.Client
 
 	smartWalletRpc *ethclient.Client
+
+	agg *Aggregator
```

Comment:
is this addition due to the access of `r.agg.chainID`? The `chainID` is more of a global config, or should live in a global context of the program.

Is it possible not to add a new reference in this RpcServer? Is `config` or `engine` a good place for global constants?  

For example, the chainID should be easily accessed band included in the `apContent` of core/taskengine/vm.go, as `apContext.chainId`.